crates/bevy_render/src/render_asset.rs

```diff
@@ -67,7 +63,8 @@ bitflags::bitflags! {
     /// If you have an asset that doesn't actually need to end up in the render world, like an Image
     /// that will be decoded into another Image asset, use `MAIN_WORLD` only.
     #[repr(transparent)]
-    #[derive(Serialize, TypePath, Deserialize, Hash, Clone, Copy, PartialEq, Eq, Debug)]
+    #[derive(Serialize, Deserialize, Hash, Clone, Copy, PartialEq, Eq, Debug, Reflect)]
```

Comment:
We should probably register these traits

Reply:
```suggestion
    #[derive(Serialize, Deserialize, Hash, Clone, Copy, PartialEq, Eq, Debug, Reflect)]
    #[reflect(Serialize, Deserialize, Hash, PartialEq, Debug, Reflect)]
```